flow/project.py

```diff
@@ -720,55 +876,68 @@ def op2(job):
         This lets options like ``--num_passes`` to be given to a group.
     :type options:
         str
+    :param flow_aggregate:
+        The operation specific aggregate parameters specified
+        using the :py:class:`aggregate`
+    :type flow_aggregate:
+        tuple
```

Comment:
The fact that the user is passing in a tuple here is a good sign I think to have an `Aggregate` class that handles all aspects of the creating aggregates from selected jobs.